portal-api/src/v1/controllers/amendments/patch-amendment.test.ts

```diff
@@ -0,0 +1,111 @@
+import { ObjectId } from 'mongodb';
+import httpMocks from 'node-mocks-http';
+import { HttpStatusCode } from 'axios';
+import { AMENDMENT_STATUS, AMENDMENT_TYPES, aPortalSessionUser, PortalFacilityAmendmentWithUkefId, TestApiError } from '@ukef/dtfs2-common';
+import { generatePortalAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import api from '../../api';
+import { patchAmendment, PatchAmendmentRequest } from './patch-amendment.controller';
+
+jest.mock('../../api');
+
+const facilityId = new ObjectId().toString();
+const amendmentId = new ObjectId().toString();
+const dealId = new ObjectId().toString();
+
+const update = { changeCoverEndDate: true };
+const user = aPortalSessionUser();
+
+describe('controllers - facility amendment', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('PATCH - patchAmendment', () => {
+    it('should call api.patchPortalFacilityAmendment with the correct parameters', async () => {
+      // Arrange
+      const { req, res } = httpMocks.createMocks<PatchAmendmentRequest>({
+        params: { facilityId, amendmentId },
+        body: { update },
+        user,
+      });
+
+      // Act
+      await patchAmendment(req, res);
+
+      // Assert
+      expect(api.patchPortalFacilityAmendment).toHaveBeenCalledTimes(1);
```

Comment:
Can we please add `console.error` to not be called.

Reply:
Done 👍